test/functional/feature_segwit.py

```diff
@@ -241,7 +241,7 @@ def run_test(self):
         tx.vin.append(CTxIn(COutPoint(int(txid1, 16), 0), b''))
         tx.vout.append(CTxOut(int(49.99 * COIN), CScript([OP_TRUE, OP_DROP] * 15 + [OP_TRUE])))
         tx2_hex = self.nodes[0].signrawtransactionwithwallet(ToHex(tx))['hex']
-        txid2 = self.nodes[0].sendrawtransaction(tx2_hex)
+        txid2 = self.nodes[0].sendrawtransaction(hexstring=tx2_hex)
```

Comment:
411391463ed9de5076f419f6451150dffea7225b

Could use named parameters only when `maxfeerate=` so these could be reverted.

Reply:
Thanks for reviewing! True, done.